src/background/providers/cloudflare.test.ts

```diff
@@ -154,7 +152,7 @@ describe('issuance', () => {
                 timeStamp: 1,
                 requestBody: {
                     formData: {
-                        ['md']: ['body-param'],
+                        /* remove 'md' parameter. */
```

Comment:
This test's comment says:
```
The request is invalid only if the body has both
         * 'md' and 'r' params.
```
Should we amend the comment? Also, what is the purpose of the test if we're not checking for both params any more?

Reply:
The amended comment for this test should be: 
`The request is invalid if the body has no 'md' parameter.`
will update it and the test. Thanks, well noted.